include/swift/SIL/SILDeclRef.h

```diff
@@ -162,6 +162,17 @@ struct SILDeclRef {
     AsyncEntryPoint,
   };
 
+  /// Represents the variants of a back deployable function.
+  enum class BackDeploymentKind : unsigned {
+    /// The original function.
+    None,
+    /// The thunk variant of a function that calls either the original function
+    /// or the fallback variant if the original is unavailable.
```

Comment:
Mention this one is @_aeic / PublicNonABI

Reply:
Updated comments